thetis/callback.py

```diff
@@ -481,3 +481,39 @@ def __call__(self):
             field_vals.append(np.reshape(self._evaluate_field(field_name), (ndetectors, -1)))
 
         return np.hstack(field_vals)
+
+
+class ScalarIntegrationCallback(DiagnosticCallback):
```

Comment:
I wonder if this could be called `AccumulatorCallback`, `CumulativeCallback`, `ScalarCumulativeCallback`, or something like that? `Integration` might be misleading as it could be confused with spatial integration.

Reply:
Good point, thanks